callback.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * IDPay payment gateway
+ *
+ * @developer JMDMahdi, meysamrazmi, vispa, Mohammad Malek(MimDeveloper.Tv)
+ * @publisher IDPay
+ * @copyright (C) 2018-2020 IDPay
+ * @license http://www.gnu.org/licenses/gpl-2.0.html GPLv2 or later
+ *
+ * https://idpay.ir
+ */
+@session_start();
+if (isset($_GET['do'])) {
+    include(dirname(__FILE__) . '/../../config/config.inc.php');
+    include(dirname(__FILE__) . '/../../header.php');
+    include_once(dirname(__FILE__) . '/idpay.php');
```

Comment:
you moved the file but I think you didn't test it after
these lines shouldn't work